### PR TITLE
ENH: adds diff-abundance/gneiss tutorial

### PR DESCRIPTION
**Pull request by ebolyen**
I corrected a couple typos from the original post, but there may be more. 

`make preview DEBUG=tutorails/gneiss` looks good.

I still need to create some data.qiime2.org links, but that doesn't need to prevent review.

### CODE REVIEW

File: source/tutorials/gneiss.rst [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+Rather than focusing on individual taxa, we can focus on the log ratios, since the log ratios are consist between the original abundances and the observed proportions of the species are observed.  The concept of calculating balances for multiple species can be extended to trees as shown in the following example.
+
+.. image:: images/gneiss-balances.jpg
+
+On the right, we define a tree, where each of the tips corresponds to a taxon and their corresponding proportions.  The internal nodes define the log ratio between the taxa underneath.  On the left, we have the same tree, but one of the taxa abundances changes.  As we have observed before, the proportions of all of the taxa will change, but looking at the balances, only the balances containing the purple taxa will change.  In this case, balance :math:`b_3` won't change, since it only considers the ratio between the red and green taxa.  By looking at balances instead proportions, we can control for the variance by restricting observations to only focus on the taxa within the balance.
```
Comment (thermokarst) [inline source/tutorials/gneiss.rst]: Can we flip the order that we discuss the images? It feels strange talking about the image on the right, then the image on the left.

File: source/tutorials/gneiss.rst [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+Where :math:`x` and :math:`y` represent the proportions of two microbes across all of the samples.   If two microbes are highly correlated, then this quantity will shrink close to zero.  Ward hierarchical cluster will then use this distance metric to iteratively cluster together groups of microbes that are correlated with each other.  In the end, the tree that we obtain will highlight the high level structure and identify the blocks within in the data.
+
+.. command-block::
+
+   qiime gneiss correlation-clustering \
+     --i-table composition.qza \
+     --o-clustering hierarchy.qza
+
+Now that we have a tree that defines our partitions, we can perform the ilr transform.  Note that the ilr transform just computes the log ratios between groups; in this case we will be computing log ratios between groups of anti-correlated features.
```
Comment (thermokarst) [inline source/tutorials/gneiss.rst]: `s/ilr/ILR/g`

File: source/tutorials/gneiss.rst (hunk 1 above)
Comment (ebolyen) [reply source/tutorials/gneiss.rst]: Turns they were discussed left-to-right, they just had the wrong side listed.

**Comment by jairideout** [comment]
I verified @thermokarst's review comments were addressed and build locally with `make preview`. Thanks @ebolyen for porting the tutorial!

**Comment by mortonjt** [comment]
Thanks!!

**Pull request merged**